tests/console/slp.pm

```diff
@@ -22,23 +22,33 @@ use strict;
 use warnings;
 use utils qw(zypper_call systemctl script_retry);
 use Utils::Systemd 'disable_and_stop_service';
+use version_utils 'is_jeos';
 
 sub run {
     my ($self) = @_;
     select_console 'root-console';
 
-    zypper_call 'in openslp openslp-server';
+    # Let's install slpd
+    zypper_call 'in openslp-server';
 
     disable_and_stop_service($self->firewall) if (script_run("which " . $self->firewall) == 0);
 
-    assert_script_run 'useradd -d /var/lib/empty/ -s /sbin/nologin openslp || true';
+    # I guess it is a bug
+    is_jeos && assert_script_run 'useradd -d /var/lib/empty/ -s /sbin/nologin openslp';
 
     systemctl 'enable slpd';
+    systemctl 'is-enabled slpd';
+    systemctl 'disable slpd';
+    systemctl 'is-enabled slpd', expect_false => 1;
```

Comment:
Why does it have to disable `slpd.service` here? I would assume that the test explicitly leaves it enabled for reboots.

Reply:
If it's not needed to be enabled, just disabling it once instead of enabling would do it.

Reply:
We do not really need to have the service up outside of this test case. The test case uses `qemu user network` instead of `tap`, therefore we are not expecting to catch any other slpds on the network. I am wondering if there is any chance to catch failing slpd service after reboot in current openQA test cases. Since the test case initially enabled slpd, I had no intention to remove it. 
I like the idea to clean up and not to pollute other successor test cases.

Reply:
Ok, so I'd prefer just disabling `slpd.service` instead of doing this dance here.